src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -96,6 +86,13 @@
 		this.maxConnections = maxConnections;
 	}
```

Comment:
Just delete this constructor and use only the one with the builder (this constructor is used only in a test so adapt the test)